### PR TITLE
chore: 🤖 add post connection workflow feature flag

### PR DESCRIPTION
**Pull request by cameronperera**
✅ Closes: ICU-10269

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-10269)

## Description

add post connection workflow feature flag and merge into LLB for post connection workflow

### CODE REVIEW

**Review by calcaide — APPROVED** [review]
The beginning of post connection workflow!! let's goo!!! 🥳

File: ui/desktop/config/environment.js [hunk 1]
```diff
@@ -85,6 +86,7 @@ module.exports = function (environment) {
       : true;
 
     // Enable features in development
+    ENV.featureFlags['post-connection-workflow'] = true;
```
Comment (calcaide) [inline ui/desktop/config/environment.js:89]: @cameronperera Shouldn't we include the flag within test environment as well?

Comment (cameronperera) [reply ui/desktop/config/environment.js:89]: We did in the past but we decided to move to enabling a feature flag in the test that directly needs it to be enabled.

Comment (calcaide) [reply ui/desktop/config/environment.js:89]: @cameronperera that is totally correct and I completly forget about it. Thanks for the reminder!

**Pull request merged**